qiskit_dynamics/arraylias_state.py

```diff
@@ -16,14 +16,36 @@
 
 from typing import Union
 from arraylias import numpy_alias
+from scipy.sparse import csr_matrix
 from .array import Array
+try:
+    from jax.experimental.sparse import BCOO
+except ImportError:
+    pass
 
 
 DYNAMICS_ALIAS = numpy_alias()
 
 # Set qiskit_dynamics.array.Array to be dispatched to numpy
 DYNAMICS_ALIAS.register_type(Array, "numpy")
 
+# register required custom versions of functions for csr type here
+DYNAMICS_ALIAS.register_type(csr_matrix, lib="scipy_sparse")
```

Comment:
Is it necessary to add other classes which is a subclass of `spmatrix`?
- DYNAMICS_ALIAS.register_type(csc_matrix, lib="scipy_sparse")
- DYNAMICS_ALIAS.register_type(coo_matrix, lib="scipy_sparse")